sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClient.cs

```diff
@@ -92,6 +92,72 @@ protected CommunicationIdentityClient()
             RestClient = null;
         }
 
+        /// <summary>Gets user details. <see cref="CommunicationUserIdentifier"/>.</summary>
+        /// <param name="communicationUser">Identifier of the identity. </param>"
```

Comment:
```suggestion
        /// <param name="communicationUserIdentifier">Identifier of the identity. </param>"
```

Reply:
I want to keep it consistent with all the other methods (ex. DeleteUser uses communicationUser)